examples/aes/src/aes.rs

```diff
@@ -146,10 +152,10 @@ fn rotate_word(w: Word) -> Word {
 
 fn slice_word(w: Word) -> Word {
     Word([
-        SBOX[U8::declassify(w[0]) as usize],
-        SBOX[U8::declassify(w[1]) as usize],
-        SBOX[U8::declassify(w[2]) as usize],
-        SBOX[U8::declassify(w[3]) as usize],
+        SBOX[declassify_usize_from_U8(w[0])],
```

Comment:
Is that a function added to `lib` ? If we have a function like that for all the cases it means ~30 functions more to add to `Hacspec.Lib` in F*, which is a pain.

Reply:
It's a function you added a while back 😉  https://github.com/hacspec/hacspec/commit/e8f667dea4ba10471c93e331cc593c538813aa27

Reply:
Ha ha ha I guess that's a boomerang then :) I'll add them to F* as I go then!